src/components/FullscreenHearing.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import {intlShape} from 'react-intl';
+import FullscreenPlugin from './FullscreenPlugin';
+import find from 'lodash/find';
+import {getHearingURL} from '../utils/hearing';
+import {userCanComment, userCanVote} from '../utils/section';
+import {Hearing, wrapHearingComponent} from './Hearing';
+
+
+class FullscreenHearing extends Hearing {
+
+  render() {
+    const hearing = this.props.hearing;
+    const user = this.props.user;
+
+    const hearingAllowsComments = !hearing.closed && (new Date() < new Date(hearing.close_at));
```

Comment:
Well spotted. I actually was going to do this, but felt the urge to rush this PR. Will refactor right away!